requirements.txt

```diff
@@ -1,7 +1,7 @@
 -e git+https://github.com/GSA/ckan.git@inventory#egg=ckan
 -e git+https://github.com/ckan/ckanext-harvest.git@13dbb1eea428e6b274f98c36bf29d42e62a70af7#egg=ckanext-harvest
 -e git+https://github.com/GSA/ckanext-datajson.git@master#egg=ckanext-datajson
--e git+https://github.com/GSA/USMetadata.git@master#egg=ckanext-usmetadata
+-e git+https://github.com/GSA/USMetadata.git@30e09a8752e795d8522f30233069e72d8242311a#egg=ckanext_usmetadata
```

Comment:
@kmanaseryan I believe here we should point to master branch again.

Reply:
@anuveyatsu agree!